picard/ui/options/interface.py

```diff
@@ -46,6 +75,11 @@ class InterfaceOptionsPage(OptionsPage):
         config.TextOption("setting", "ui_language", u""),
         config.BoolOption("setting", "starting_directory", False),
         config.TextOption("setting", "starting_directory_path", ""),
+        config.ListOption("setting", "toolbar_layout", [
+            'add_directory_action', 'add_files_action', 'seperator', 'cluster_action', 'seperator',
```

Comment:
AFAIK, correct spelling is "separator" not "seperator", many occurences in this PR